app.js

```diff
@@ -25,8 +25,9 @@ app.use(
     saveUninitialized: false
   })
 );
-
+app.get('*', (req, res) => res.status(404).send(
```

Comment:
You can take this catch all route handler out for now

Reply:
ok. will do that.